tests/TestCase/ORM/QueryTest.php

```diff
@@ -2993,22 +2993,21 @@ public function testLeftJoinWithNested()
 
         $results = $table
             ->find()
-            ->select(['total_articles' => 'count(articles.id)'])
+            ->select([
+                'authors.id',
+                'total_articles' => 'count(tags.id)'])
             ->leftJoinWith('articles.tags', function ($q) {
                 return $q->where(['tags.name' => 'tag3']);
             })
-            ->autoFields(true)
-            ->group(['authors.id', 'authors.name']);
+            ->group(['authors.id']);
 
         $expected = [
-            1 => 2,
+            1 => 0,
```

Comment:
I changed this test so that it does not fail in mysql 5.7 with the new `group by` mode